clara/plugins/clara_images.py

```diff
@@ -441,13 +458,19 @@ def system_install(work_dir, dist):
                            work_dir)
 
     # Manage groupinstall for centos
-    if dists[ID]['pkgManager'] == "yum":
+    if dists[ID]['pkgManager'] == "dnf":
         group_pkgs = get_from_config("images", "group_pkgs", dist)
         if len(group_pkgs) == 0:
             logging.warning("group_pkgs hasn't be set in the config.ini")
         else:
             group_pkgs = group_pkgs.split(",")
             run_chroot(["chroot", work_dir, distrib["pkgManager"], "groupinstall", "-y", "--nobest"] + group_pkgs, work_dir)
+            # Remove original repo as we are in air-gapped environment and so haven't outside internet access!
+            dir_path = os.path.dirname(os.path.realpath(src_list))
+            files = glob.glob(dir_path+"/?%s-*.repo" % ID[1:])
+            for f in files:
+                logging.debug("removing repo file %s" % f)
+                os.remove(f)
```

Comment:
This code block is duplicated, a function to factorize it would be better for maintenance.